### PR TITLE
Issue/368 spike shading for graph

### LINKED ISSUE #368: Spike: Shading Graph
- test graphing library can do shading
- produce graph/screen shot of shading on our UI. Use 90% of value, and 110% of value for the upper and lower bounds

### PR DESCRIPTION
**Pull request by rachel-labri-tipton**
# Pull Request

## Description
Sees if the Quartz Solar chart library, Recharts, is up to the task of shading probabilistic forecasts. For each forecast, the shading was supposed to capture 110% and 90% of the OCF forecast value with the same timestamp. I think it should be fairly easy to do this with the Recharts library. 

Recharts can shade an area between two values input as a range. Here's an example:

Screenshot 2023-06-21 at 12 16 50

To try this out, I added `probabilisticRangeData` to the `use-format-chart-data` function which prepares forecast data for the national forecast chart. I added a `PROBABILISTIC_RANGE` datakey to `remix-line.tsx`. I added the probabilistic data in the chart as an `<Area/>` component. Here's how it looks with `#7BCDF3`:

Screenshot 2023-06-21 at 12 20 48

Fixes #368

## How Has This Been Tested?

Code was tested by running locally and looking at the chart. 
- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by peterdudfield** [comment]
Amazing thats really great to test this out. 
Thank you for doing this

What do you think about merging / not merging? 
- merging means this code is always there, but perhaps makes it less clean
- not merging keeps the code clean, and then this is always in the git history,
- you should share this on slack on progress, just to show these nice shadings can be done

**Comment by rachel-labri-tipton** [comment]
hi @peterdudfield,  I wanted to share what the tooltip looks like with the `plevels` added in. Is this what you were envisioning? I also was able to get the dots we were looking at yesterday to show up as yellow. 

Screenshot 2023-07-11 at 14 56 50

File: apps/nowcasting-app/components/charts/remix-line.tsx [hunk 1]
```diff
@@ -345,6 +349,46 @@ const RemixLine: React.FC<RemixLineProps> = ({
               />
             </>
           )}
+
+          <Line
```
Comment (peterdudfield) [inline apps/nowcasting-app/components/charts/remix-line.tsx:353]: What are these two new lines?

**Review by peterdudfield — APPROVED** [review]
Looks great, ive left a comment, but its more of a question

File: apps/nowcasting-app/components/charts/remix-line.tsx (hunk 1 above)
Comment (rachel-labri-tipton) [reply apps/nowcasting-app/components/charts/remix-line.tsx:353]: thanks for spotting these. the order of the lines getting slotted into the graph dictates how the lines appear. so I'll delete one set of the lines.

**Comment by rachel-labri-tipton** [comment]
hey @peterdudfield, just pushed a few changes and this should be ready for `dev`? 
Screenshot 2023-07-13 at 11 05 52

**Pull request merged**